docs/cue-lists/creating-a-cue-list.md

```diff
@@ -190,46 +201,42 @@ times set for the cue, but you can also select \[Use Individual Target
 Times\] which will use the times set in the autoloaded playback, or you
 can set independent times yourself.
 
-Running a key macro from a cue list
+Running a Key Macro from a Cue List
 -----------------------------------
 
 You can program a cue within a cue list to trigger a key macro. This
 allows you to set up custom actions which can be performed when a cue in
 a cue list fires; for example you may want to lamp on your fixtures as
 the first cue in the cue list.
 
-1. Press \[Playback Options\].
+1. Press \[Playback Options\]
 
-2. Select the cue list.
+2. Select the **cue list**
 
-3. Select \[Macros\].
+3. Select \[Macros\]
 
-4. In the Playback View window, select the step you want to add a macro
-to.
+4. In the **Playback View window**, select the **step** you want to add a macro
+to
 
-5. Press the button(s) for the macro(s) you want to attach to the step.
-Or press \[Add\] and choose an action from the list.
+5. Press the button(s) for the macro(s) you want to attach to the step,
+or press \[Add\] and choose an action from the list.
 
-3. The "Macros" column shows the macros you have added.
+6. The **"Macros"** column shows the macros you have added.
 
--   You can remove macros by selecting the cue, pressing the softkey for
+> You can remove macros by selecting the cue, pressing the softkey for
     the macro you wish to remove and then pressing \[Remove Link\].
 
-Keyboard shortcuts/syntax for cue lists
+Keyboard Shortcuts/Syntax for Cue Lists
 ---------------------------------------
 
 The following keyboard syntax can be used for fast cue list recording
 and editing. These commands work for the selected or currently connected
-playback where n is the cue number.
-
--   \<Record\>, \<Connect\>, n, \<Enter\> (Record cue n)
-
--   \<Copy\>, \<Connect\>, n (Copy cue n)
-
--   \<Delete\>, \<Connect\>, n (Delete cue n)
-
--   \<Include\>, \<Connect\>, n (Include cue n)
-
--   \<Connect\>, n, \<Go\>. (Go cue n)
-
-
+playback where **n** is the cue number.
+
+Shortcut | Action
+--- | ---
```

Comment:
You can format these with extra hypens and spaces so that they display nice in the raw file